backend/core/src/seeder/seeds/applications.ts

```diff
@@ -13,53 +13,20 @@ import { Application } from "../../applications/entities/application.entity"
 import { ApplicationsService } from "../../applications/services/applications.service"
 import { ApplicationCreateDto } from "../../applications/dto/application-create.dto"
 
-const applicationCreateDtoTemplate: Omit<
-  ApplicationCreateDto,
-  "user" | "listing" | "listingId" | "preferredUnit"
-> = {
-  acceptedTerms: true,
-  accessibility: {
-    hearing: false,
-    mobility: false,
-    vision: false,
-  },
-  additionalPhone: false,
-  additionalPhoneNumber: undefined,
-  additionalPhoneNumberType: undefined,
-  alternateAddress: {
-    city: "city",
-    county: "county",
-    latitude: 52.0,
-    longitude: 50,
-    placeName: "Place Name",
-    state: "state",
-    street: "street",
-    street2: "street2",
-    zipCode: "zip code",
-  },
-  alternateContact: {
-    agency: "agency",
-    emailAddress: "test@example.com",
-    firstName: "First",
-    lastName: "Last",
-    mailingAddress: {
-      city: "city",
-      county: "county",
-      latitude: 52.0,
-      longitude: 50,
-      placeName: "Place Name",
-      state: "state",
-      street: "street",
-      street2: "street2",
-      zipCode: "zip code",
+const getApplicationCreateDtoTemplate = (
+  jurisdictionString: string
```

Comment:
The diff is odd here, the only difference is the added `jurisdictionString` prop